pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -964,3 +979,12 @@ func (tc *TidbCluster) WithoutLocalTiDB() bool {
 func (tc *TidbCluster) AcrossK8s() bool {
 	return tc.Spec.AcrossK8s
 }
+
+func (tc *TidbCluster) StartScriptVersion() StartScriptVersion {
```

Comment:
how about:

```go
  func (tc *TidbCluster) StartScriptVersion() StartScriptVersion {
      if tc.Spec.StartScriptVersion == "" {
          return StartScriptV1
      }

      return tc.Spec.StartScriptVersion
  }
```

Reply:
For non-supported configurations, the current code will return v1.